mk/build.mk

```
@@ -1,16 +1,7 @@
-BUILD_INFO_GIT_TAG ?= $(shell git describe --tags 2>/dev/null || echo unknown)
-BUILD_INFO_GIT_COMMIT ?= $(shell git rev-parse HEAD 2>/dev/null || echo unknown)
-BUILD_INFO_BUILD_DATE ?= $(shell date -u +"%Y-%m-%dT%H:%M:%SZ" || echo unknown)
-BUILD_INFO_VERSION ?= $(shell $(TOOLS_DIR)/releases/version.sh)
-BUILD_INFO_ENVOY_VERSION ?= $(ENVOY_VERSION)
-
-build_info_fields := \
-	version=$(BUILD_INFO_VERSION) \
-	gitTag=$(BUILD_INFO_GIT_TAG) \
-	gitCommit=$(BUILD_INFO_GIT_COMMIT) \
-	buildDate=$(BUILD_INFO_BUILD_DATE) \
-	Envoy=$(BUILD_INFO_ENVOY_VERSION)
-build_info_ld_flags := $(foreach entry,$(build_info_fields), -X github.com/kumahq/kuma/pkg/version.$(entry))
+build_info := $(shell $(TOOLS_DIR)/releases/version.sh)
+BUILD_INFO_VERSION ?= $(firstword $(build_info))
```

```suggestion
BUILD_INFO_VERSION ?= $(word 1 $(build_info))
```

I find this easier to follow as the second line uses `word` again